derive_builder/src/lib.rs

```diff
@@ -216,6 +216,31 @@
 //! }
 //! ```
 //!
+//! ## Fallible Setters
+//!
+//! Alongside the normal setter methods, you can expose fallible setters which are generic over
+//! the `TryInto` trait. TryInto is a not-yet-stable trait 
+//! (see rust-lang issue [#33417](https://github.com/rust-lang/rust/issues/33417)) similar to
+//! `Into` with the key distinction that the conversion can fail, and therefore produces a `Result`.
+//! 
+//! You can declare the `try_setter` attribute today, and your builder will work on stable Rust -
+//! the fallible setters will automatically light up in future when the `try_from` feature is
+//! stabilized.
+//!
+//!
+//! ```rust,ignore
+//! # #[macro_use]
+//! # extern crate derive_builder;
+//! #
+//! #[derive(Builder, Debug, PartialEq)]
+//! #[builder(try_setter, setter(into))]
+//! struct Lorem {
+//!     pub ipsum: u8,
+//! }
+//! ```
+//!
+//! A complete example of using fallible setters is available in `examples/try_setter.rs`.
```

Comment:
please add a note that the name and visibility are inherited from the `setter` settings.

You could change the example into something like this, to better explain it?
```rust
//! # #[macro_use]
//! # extern crate derive_builder;
//! #
//! #[derive(Builder, Debug, PartialEq)]
//! struct Lorem {
//!     #[builder(try_setter, setter(name="foo"))]
//!     pub ipsum: u8,
//! }
```